dash/dash-renderer/src/actions/callbacks.ts

```diff
@@ -508,8 +512,37 @@ function handleServerside(
                     }
 
                     if (!long || data.response !== undefined) {
-                        completeJob();
-                        finishLine(data);
+                        const newLibs: string[] = [];
+                        Object.values(data.response as any).forEach(
+                            (newData: any) => {
+                                Object.values(newData).forEach(newProp => {
+                                    crawlLayout(newProp, (c: any) => {
+                                        if (!libraries.includes(c.namespace)) {
```

Comment:
minor perf point - wouldn't it be better to leave `libraries` as a map?
```js
const {libraries} = getState();
...
if (!(c.namespace in libraries))
```

Reply:
I think the perf gain would be minimal here since the number of libraries should never be high, I added another condition that was missing and I think it's more readable if both conditions are the same.